src/schema/spell.ts

```diff
@@ -56,6 +56,12 @@ export interface SpellData {
      */
     range: number | string;
 
+    /**
+     * The area of effect of the spell.
+     * example: 30ft cube
+     */
+    aoe: string;
```

Comment:
should this be optional?

Reply:
Yup I just realized that and went to check the PR lol